pepper-apis/src/main/resources/db-changes/schema/DDP-6520-singular-add-operator-id-to-age-up-candidate.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="ypavlov" id="20211006-add-operator_user_id-to-age_up_candidate">
```

Comment:
Where can I find details / requirement on why we need this schema change. 
Looks like adding operator needed changes to several AgeUp related code.